feature/session/src/main/java/io/github/droidkaigi/confsched2020/session/ui/item/SessionDetailDescriptionItem.kt

```diff
@@ -23,7 +23,7 @@ import io.github.droidkaigi.confsched2020.session.databinding.ItemSessionDetailD
 class SessionDetailDescriptionItem @AssistedInject constructor(
     @Assisted private val session: Session
 ) :
-    BindableItem<ItemSessionDetailDescriptionBinding>() {
+    BindableItem<ItemSessionDetailDescriptionBinding>(1L) {
```

Comment:
Thank you for checking behavior 👍 
My recommendation for this problem.

```
override fun isSameAs(other: Item<*>?): Boolean = other is SessionDetailDescriptionItem
```

Reply:
Thank you for telling Groupie usage.
I applied this comment.